src/EditorFeatures/VisualBasicTest/QuickInfo/SemanticQuickInfoSourceTests.vb

```diff
@@ -106,25 +106,25 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.QuickInfo
         <Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)>
         Public Async Function TestInteger() As Task
             Await TestInClassAsync("Dim i As $$Integer",
-             MainDescription("Structure System.Int32",
-              ExpectedClassifications(
-               Keyword("Structure"),
-               WhiteSpace(" "),
-               Identifier("System"),
-               Operators.Dot,
-               Struct("Int32"))))
+                MainDescription(
+                    ("Structure", TextTags.Keyword),
+                    (" ", TextTags.Space),
+                    ("System", TextTags.Namespace),
+                    (".", TextTags.Operator),
+                    ("Int32", TextTags.Struct)
+                ))
```

Comment:
As I mentioned above, the previous `expectedClassifications` argument was unused and not actually tested. I instead added a way to verify the `TextTag`s and rewrote existing tests to use it. I believe testing `TextTag` is the right thing to do because that's what's returned by the actual public quick info API, and converting it to classifications loses some information (for example locals & parameters, fields, methods are all identifier).